policyengine_us/variables/household/cliff.py

```diff
@@ -10,9 +10,15 @@ class cliff_evaluated(Variable):
 
     def formula(person, period, parameters):
         adult_index_values = person("adult_index", period)
-        cliff_adult_count = parameters(period).simulation.cliff_adults
+        mtr_adult_count = parameters(
+            period
+        ).simulation.marginal_tax_rate_adults
         is_adult = person("is_adult", period)
-        return is_adult & (adult_index_values <= cliff_adult_count)
+        with open("test_file.txt", "a") as f:
```

Comment:
can we remove this?

Reply:
Ugh, I hadn't meant to push that. Thanks for checking, let me look over all the changes with fresh eyes and get this merge-ready.